main.py

```diff
@@ -19,6 +19,89 @@
 def hello():
     return 'Hello, Circles!!'
 
+# User API
+
+
+@app.route('/circles/api/v1.0/users/<id>', methods=['GET'])
```

Comment:
Thanks for doing these, it looks good! Could you submit a PR with just this endpoint and none of the commented out ones? You can keep those changes locally but let's keep prod clean I think. Then, when you're adding API stubs we can submit them as PR's one by one so they're manageable to review (prefer not to merge a big PR with all the stubs at once). Thanks!